spec/features/admin_deletes_item_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+feature "admin deletes an item" do
+
+  let(:user) do
+    User.create(
+      email: "pinkpinksopink@gmail.com",
+      password: "123123123",
+      role: "admin"
+    )
+  end
+
+  let(:item) do
+    Item.create(
+      item_url: "http://www.ikea.com/us/en/catalog/products/80176284/",
+      title: "HEMNES",
+      subtitle: "Coffee table, black-brown",
+      picture_url: "http://www.ikea.com/us/en/images/products/hemnes-coffee-table-brown__0104030_PE250678_S4.JPG",
+      price: "139.00"
+    )
+  end
+
+  scenario "admin successfully deletes an item" do
+    user
+    item
```

Comment:
If you use `let!` instead of `let`, it will actually run the `:user` and `:item` blocks before each test *even if you don't call them*, so you can remove lines 24 and 25 by using `let!(:user)`, etc